src/openzaak/components/autorisaties/tests/admin/test_autorisaties_admin.py

```diff
@@ -122,28 +124,31 @@ def _add_autorisatie(self, obj, **kwargs):
 
 
 @tag("admin-autorisaties")
-class ManageAutorisatiesAdmin(NotificationsConfigMixin, TransactionTestCase):
-    def setUp(self):
-        super().setUp()
+class ManageAutorisatiesAdmin(NotificationsConfigMixin, TestCase):
```

Comment:
I'm sorry for this refactor, but this test case was horribly slow making TDD a pain :(